R/system.R

```diff
@@ -339,7 +339,7 @@ js_glue_transformer <- function(expr, envir) {
     }
   }
 
-  if (is.null(val)) { # NULL in R is same as empty vector. Print empty string.
+  if (length(val) == 0) { # Empty vector case. NULL in R is same as empty vector, but since is.null(as.character(c(0))) returns FALSE, this has to be done my length().
```

Comment:
Do you mean "by length()" by "my length()"?

Reply:
That's right. Fixed it. Thanks!